src/benchmark/tpch_data_micro_benchmark.cpp

```diff
@@ -0,0 +1,208 @@
+#include "micro_benchmark_basic_fixture.hpp"
+
+#include "expression/expression_functional.hpp"
+#include "operators/join_hash.hpp"
+#include "operators/join_sort_merge.hpp"
+#include "operators/table_scan.hpp"
+#include "operators/table_wrapper.hpp"
+#include "storage/chunk_encoder.hpp"
+#include "storage/encoding_type.hpp"
+#include "storage/storage_manager.hpp"
+#include "tpch/tpch_db_generator.hpp"
+
+using namespace opossum::expression_functional;  // NOLINT
+
+namespace opossum {
+
+class TableWrapper;
+
+// Defining the base fixture class
+class TPCHDataMicroBenchmarkFixture : public MicroBenchmarkBasicFixture {
+ public:
+  void SetUp(::benchmark::State& state) {
+    auto& sm = StorageManager::get();
+    const auto scale_factor = 0.001f;
+
+    if (!sm.has_table("lineitem")) {
+      std::cout << "Generating TPC-H data with scale factor " << scale_factor << " ... " << std::flush;
+      TpchDbGenerator(scale_factor, 100'000).generate_and_store();
```

Comment:
The default parameter for the chunk size is `Chunk::DEFAULT_SIZE` which is `100'000`. You could not specify it at all or use `Chunk::DEFAULT_SIZE`.